projects/robots/abb/irb/worlds/inverse_kinematics.wbt

```diff
@@ -85,13 +89,10 @@ Robot {
           ]
           leadSize 0.01
           maxDistance 0.05
-          write FALSE
         }
       ]
     }
   ]
-  controller "inverse_kinematics"
```

Comment:
I would set the controller to `""` to avoid running a useless void controller.

Reply:
Except this everything is working perfectly.

Reply:
You're right, but surprisingly this features does not runs smoothly: because of this https://github.com/cyberbotics/webots/issues/1047

=> I propose to merge this PR as-is, and to come back on this with a patch to #1047.